### DISCUSSION TITLE
New forms show "filament-support::components.grid" instead of the content...

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#4879] [Help] [answered]
Weird issue, but my new forms show "filament-support::components.grid" instead of the content it is supposed to have. All my other forms are ok and work perfectly fine in local, but once it's on my dev server, this is what is shown...

Did I forget a step for it to show that instead of the content?

**Comment by NicolasSacC** [comment, marked as answer]
Well, for future reference, you just have to clear the view cache...

**Reply by danharrin** [reply]
Make sure `filament:upgrade` is in your deploy script
